cds/config.py

```diff
@@ -697,6 +729,8 @@ def _(x):
 
 # The site name
 THEME_SITENAME = _('CERN Document Server')
+# Default site URL (used only when not in a context - e.g. like celery tasks).
+THEME_SITEURL = "https://videos.cern.ch"
```

Comment:
I would leave that one for puppet, is it use for anything else besides building the URL?

Maybe we could add by default `http://localhost:5000`

Reply:
Yeah, I put it here just to have the default value and puppet is overwriting it. I can change it to `localhost`